support-frontend/assets/components/headers/header/header.scss

```diff
@@ -253,12 +253,12 @@ $header-links-min-width: gu-span(12);
         color: gu-colour(neutral-100);
         @include gu-fontset-body-sans;
         border-right: 1px solid gu-colour(brand-pastel);
-  
+
         @include mq($until: tablet) {
           flex: none;
           border-right: none;
         }
-      
+
```

Comment:
which editor setting removes these spaces? I haven't seen much of this kind of change, but might be worth us all making this setting on to avoid this noise in future. @lucymonie any idea?
( from https://github.com/guardian/support-frontend/pull/1783/files#diff-a03618d9ea75a35d75993c4b81e6049cR261 )